cms/static/css/nhsuk.css

```diff
@@ -8751,8 +8751,16 @@ nav a:hover, nav a:visited {
   border: 0px;
 }
 
-#mega-menu {
+.toggle-content {
   display: none;
```

Comment:
This is the line I was talking about losing to see if we gain the performance back (if needed), but for the moment we should leave it in.